### ISSUE TITLE
I found this project from your paper. I want to know whether the code is public?

### ISSUE THREAD

**Issue by shoukailiang** [MohamedKari/transformr#1]
Hi， @MohamedKari .
I found this project from your paper. I want to know whether the code is public?

**Comment by herene** [comment]
yes, it is a very interesting project. Could you please  make the code public for us to learn?

**Comment by MohamedKari** [comment]
Yes, we will open-source the project!

**Comment by emmasteimann** [comment]
@MohamedKari Any updates on when you might be open sourcing this project? :-)

**Comment by hacker-jerry** [comment]
Hi, @MohamedKari , I will appreciate for your opening this paper's code, thank😊

**Comment by Nemo1999** [comment]
Hi, I would appreciate if you can provide the code for this project !
